Makefile

```diff
@@ -75,20 +81,29 @@ update-requirements: ## Updates requirement
 .PHONY: test
 test: ## Run tests
 	$(VIRTUALENV)/bin/pytest --disable-warnings -v --cov=grants_tagger
+	$(VIRTUALENV)/bin/tox
 
 
 .PHONY: build
 build: ## Create wheel distribution
 	$(VIRTUALENV)/bin/python setup.py bdist_wheel
+	tar -c -z -v -f models/$(MESH_MODEL_PACKAGE) models/$(MESH_MODEL) models/$(MESH_LABEL_BINARIZER)
 
 .PHONY: deploy
 deploy: ## Deploy wheel to public s3 bucket
 	aws s3 cp --recursive --exclude "*" --include "*.whl" --acl public-read dist/ s3://$(PUBLIC_PROJECT_BUCKET)
 	git tag v$(shell python setup.py --version)
 	git push --tags
 	$(VIRTUALENV)/bin/python -m twine upload --repository pypi dist/*
-	tar -c -z -v -f models/$(MESH_MODEL).tar.gz models/$(MESH_MODEL) models/$(MESH_LABEL_BINARIZER)
-	gh release upload v$(shell python setup.py --version) models/$(MESH_MODEL).tar.gz
+	aws s3 cp --acl public-read models/$(MESH_MODEL_PACKAGE) s3://datalabs-public/grants_tagger/models/$(MESH_MODEL_PACKAGE)
```

Comment:
I think it's ok to do it like this for now, and fir future release we'll find better ways